.teamcity/_self/PluginBaseBuild.kt

```diff
@@ -59,6 +59,20 @@ open class PluginBaseBuild : Template({
 		bashNodeScript {
 			name = "Prepare environment"
 			scriptContent = """
+				# Merge the trunk branch first. This way, our builds and tests
+				# include the latest merged version of the plugin being built.
+				# Otherwise, we can get into a situation where the current plugin
+				# build appears "different", but that's just because it's older.
+				if [[ "%teamcity.build.branch.is_default%" != "true" ]] ; then
+					# git operations will fail if no user is set.
+					git config --local user.email "tcbuildagent@example.com"
+					git config --local user.name "Teamcity Build Agent"
+					git checkout -b tc-temp-branch
+					git merge trunk
```

Comment:
Does the script (and therefore, the build) fail if this merge is not successful?

Maybe we should capture that error and show a nice message in the logs.

Reply:
Good idea!

Reply:
This is what it looks like now: (see https://teamcity.a8c.com/buildConfiguration/calypso_WPComPlugins_EditorToolKit/6472835)

<img width="1809" alt="Screen Shot 2021-07-26 at 12 26 09 PM" src="https://user-images.githubusercontent.com/6265975/127047002-2662cd5d-dc03-4780-8f03-829c2ae3f8a1.png">